radis/api/hitempapi.py

```diff
@@ -113,6 +117,57 @@ def get_last(b):
     return b[non_zero]
 
 
+def login_to_hitran():
+    login_url = 'https://hitran.org/login/'
+    username = os.getenv('HITRAN_USERNAME') or input("Enter HITRAN username: ")
+    password = os.getenv('HITRAN_PASSWORD') or input("Enter HITRAN password: ")
+    
+    session = requests.Session()
+    response = session.get(login_url)
+    soup = BeautifulSoup(response.text, 'html.parser')
+    csrf = soup.find('input', {'name': 'csrfmiddlewaretoken'})['value']
+    
+    login_data = {
+        'csrfmiddlewaretoken': csrf,
+        'email': username,
+        'password': password,
+    }
+    
+    headers = {
+        'Referer': login_url,
+        'Origin': 'https://hitran.org',
+        'Cookie': f'csrftoken={csrf}'
+    }
+    
+    login_response = session.post(login_url, data=login_data, headers=headers, allow_redirects=False)
+    
+    if login_response.status_code == 302:
+        print("Login successful.")
+        return session
+    else:
+        print(f"Login failed: {login_response.status_code}")
+        print("Response:", login_response.text[:500])
+        return None
+
+def download_hitemp_file(session, file_url, output_filename):
+    file_response = session.get(file_url, stream=True)
+    
+    if file_response.status_code == 200:
+        total_size = int(file_response.headers.get('content-length', 0))
+        with open(output_filename, 'wb') as f:
+            downloaded = 0
+            for chunk in file_response.iter_content(chunk_size=8192):
+                if chunk:
+                    f.write(chunk)
+                    downloaded += len(chunk)
+                    done = int(50 * downloaded / total_size)
+                    print(f'\rProgress: [{"=" * done}{" " * (50-done)}] {downloaded}/{total_size} bytes', end='')
+        print("\nDownload complete!")
+    else:
+        print(f"Download failed: {file_response.status_code}")
+        print("Response:", file_response.text[:500])
```

Comment:
Same as above:

- As everywhere in RADIS, there should be a `if verbose: print('...')`
- Explicit Warning/Error messages